rules.java

```diff
@@ -19,13 +19,27 @@
 shippable:
 	mkdir -p shippable
 
+ifneq ($(IS_PULL_REQUEST),true)
+
 sonar: $(BEFORE_SONAR) sonarconfig buildreports
 	$(TOOLCHAINDIR)/tools/pullanalize
 
+createdocs: $(MODEL_BASENAME).compiled codedocs
+
+else #IS_PULL_REQUEST
+
+sonar:
+
+createdocs:
+	$(TOOLCHAINDIR)/tools/noPullRequest
```

Comment:
noPullRequest will run when $IS_PULL_REQUEST is true? Sounds weird.

Reply:
it was an ifNdef

Reply:
Add more meaningful name